postprocessors/sandbox/widebeam_downsample2normalscan.py

```diff
@@ -81,49 +81,23 @@ def __init__(self, infile: str, outfile: str, infile_type: str, outfile_type: st
         outfile_structure: str
             Borealis structure of output file. Either 'array', 'site', or 'dmap'.
         """
-        self.infile = infile
-        self.outfile = outfile
-        self.infile_type = infile_type
-        self.infile_structure = infile_structure
-        self.outfile_type = outfile_type
-        self.outfile_structure = outfile_structure
-
-
-        def check_args(self):
-
-            if self.infile_structure not in rs.FILE_STRUCTURE_MAPPING[self.infile_type]:
-                raise conversion_exceptions.ImproperFileStructureError(
-                    f'Input file structure "{self.infile_structure}" is not compatible with input file type '
-                    f'"{self.infile_type}": Valid structures for {self.infile_type} are '
-                    f'{rs.FILE_STRUCTURE_MAPPING[self.infile_type]}'
-                )
-            if self.outfile_structure not in rs.FILE_STRUCTURE_MAPPING[self.outfile_type]:
-                raise conversion_exceptions.ImproperFileStructureError(
-                    f'Output file structure "{self.outfile_structure}" is not compatible with output file type '
-                    f'"{self.outfile_type}": Valid structures for {self.outfile_type} are '
-                    f'{rs.FILE_STRUCTURE_MAPPING[self.outfile_type]}'
-                )
-            if self.infile_structure not in ['array', 'site', 'dmap']: #added to accept dmap input
-                raise conversion_exceptions.ConversionUpstreamError(
-                    f'Input file structure "{self.infile_structure}" cannot be reprocessed into any other format.'
-                )
-        check_args(self)
-        self.averaging_method = None
-        self._temp_files = []
+        super().__init__(infile, outfile, infile_type, outfile_type, infile_structure, outfile_structure)
```

Comment:
One other thing I forgot about last review - since this class only supports processing `rawacf -> rawacf`, it is best to remove the `infile_type` and `outfile_type` arguments to the `__init__` function, since those must always be `rawacf` and `rawacf`, respectively. Then, when calling `super.__init__()`, you can hard-code `rawacf` in place of `infile_type` and `outfile_type`.
```suggestion
        super().__init__(infile, outfile, "rawacf", "rawacf", infile_structure, outfile_structure)
```